indexclient/client.py

```diff
@@ -116,6 +115,24 @@ def bulk_request(self, dids):
             for doc in response.json()
         ]
 
+    def bulk_get_latest(self, dids, skip_null=False):
+        """
+        bulk get latest versions
+        """
+        headers = {'content-type': 'application/json'}
+        try:
+            response = self._post("bulk/documents/latest", params={"skip_null": skip_null},
+                                  json=dids, headers=headers)
+        except requests.HTTPError as exception:
+            if exception.response.status_code == 404:
+                return None
+            else:
+                raise exception
+        return [
+            Document(self, doc['did'], json=doc)
+            for doc in response.json()
+        ]
+
```

Comment:
Maybe now is a good time to add a little refactor here? Several of these functions follow the same `try/except/return` model.

Reply:
@anmaxvl just exposed that the try catch 404 doesn't even apply for bulk, and passing in an empty list of dids will make indexclient raise exception, instead of returning empty list of doc. I think this is a problem outside of the scope of this PR